### DISCUSSION TITLE
Fully downloaded videos not playing

### DISCUSSION THREAD

**Discussion by mfrobins212** [aclap-dev/video-downloadhelper#861] [Help and Q&A]
### Your question

Some the videos are not being recorded correctly maniny from sites like Justfor.fans, myvidster, etc. It takes up space on my hard drive like its the full video stored. However when you play on any video player, its scrambled and doesn't have no audio. I tried converting it to .avi, .mp4, .mpeg4, .mov, and .wmv. 

Is there a way to view the downloaded video or is there a fix for this issue? 

### Addon & CoApp version and details

version: 9.0.2.3
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"c4e88b8e8b624895"}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "/Users/michaelrobinson/dwhelper"
        }
    },
    "view_options": {
        "all_tabs": true,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 1,
    "successfull_dl": 2
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

Search this forum using "drm" as your search key.
